### DISCUSSION TITLE
Validator Advertisement Space

### DISCUSSION THREAD

**Discussion by markopoloparadox** [capsule-corp-ternoa/ternoa-hub#2] [General]
Hello :)

In this discussion you can advertise your node and the information that you have provided will be displayed on a soon™ website.

Rules:
- One Message per github account (or node operator)
- Any messages that are not an advertisement will be deleted. If you have a questions use #8  space for that.
- The message needs to be formatted correctly. Check the format below

Format:
- Node Addresses: _Your Node Address(es)_
- Node Names: _Your Node Names_
- Description in English (max 512 characters): _Lorem Ipsum_
- Descriptif en Français (max 512 characters) <- This is optional: _Lorem Ipsum_
- Useful links (like webpage, telegram group, etc...): _My webpage: www.tra...._

**Comment by WyvernNE** [comment]
# Node address: 
5H5zYAUc37j9MzKhjN5drx2UMDATTcyTQ39XEAS6b6Jd7cSo
# Node Name: 
Crystal-Network
# Description: 
Crystal-Network is a french-speaking community whose values are learn, educate share and respect.
What we want to achieve with our node (singular, we don't want to open multiple nodes to make this as decentralized possible) is to have an operational subject for our future educational content. 
We do not want people to only nominate our community-node, we want them to also learn more on how it works, and also have the possibility to propose enhancement.
# Descriptif: 
Crystal-Network est une communauté francophone dont les valeurs sont l'apprentissage, l'éducation, le partage et le respect.
Ce que nous voulons réaliser avec notre nœud (singulier, nous ne voulons pas ouvrir plusieurs nœuds pour rendre cela aussi décentralisé que possible) est d'avoir un sujet opérationnel pour notre futur contenu éducatif. 
Nous ne voulons pas que les gens se contentent de nommer notre nœud communautaire, nous voulons qu'ils en apprennent davantage sur son fonctionnement et qu'ils aient la possibilité de proposer des améliorations

# Useful link:
  - Discord: https://discord.gg/9hchtTsEaW
  - Mail: Hello@crystal-network.ch
  - Twitter: @Crystal_Netw0rk

**Comment by Orius** [comment]
### Node adresses:
5H8miAPvJLkgVBbQpaTgq1s45CqSK5ztjtE3iJvZPo8pqaq6
5DjnVuRmNvqFmrokATt7RGEeHdwYXoq1gyaYVT1NbQyAB39S
5EtF7pEziw7xwvGZqftjRf3GH4HFGoejXqYmGzGyxWi8Yr7B
5EcAqQ8KfS2b5TtEKEe4EBeH2humTvnMdRkjqTB3NPT2Kws2

### Node names (respectively):
ORIUS INTERNATIONAL 01
ORIUS INTERNATIONAL 02
ORIUS INTERNATIONAL 03
ORIUS INTERNATIONAL 04

### Description in English:
Our company offers those who do not have the technical skills to participate in securing the Ternoa blockchain, whose NFT concept is revolutionary and particularly promising.
To do this, our dedicated servers are distributed among different providers in different datacenters. They have capacities higher than Ternoa's requirements and are monitored 24/7. We have failovers in case of incident.
In order not to compromise the decentralization of the network, we will never exceed 5 Ternoa validator nodes.

### Descriptif en Français ;
Notre entreprise propose à ceux qui n'ont pas les compétences techniques de participer à la sécurisation de la blockchain Ternoa, dont le concept de NFT est révolutionnaire et particulièrement prometteur.
Pour ce faire, nos serveurs dédiés sont répartis chez différents fournisseurs dans différents datacenters. Ils ont des capacités supérieures aux prérequis Ternoa et sont monitorés H24 7/7. Nous disposons de serveurs de secours en cas d'incident.
Afin de ne pas compromettre la décentralisation du réseau, nous ne dépasserons jamais 5 nodes validateurs Ternoa.

### Useful links:
- Mail: contact@orius.bz
- Twitter: @OriusLLC
- Telegram: https://t.me/OriusLLC

**Comment by Crypte** [comment]
### Nodes adresses :
5ECM7nUB5VCuTBGa9V5xeCECr4CDr7d3M4gGK4DLfzCvuBBD
5EeeRBQSzx5mmtT5EjPmfEj21GbNRAGy71Q1Q3UTYSuk5Gy4
5HnLknMoTe9sLRd5bNjf9ZwYiZgtaRpgqYMjF6kSduPPxa97

### Nodes names :
CRYPTENODE - 01
CRYPTENODE - 02
CRYPTENODE - 03

### Description in English :

Passionate about the Ternoa universe since the PS2 and validator since the first blocks of the Testnet, I now offer 3 validators on the mainnet. My validators are monitored (24/24h, 7/7D), secured by me and updated regularly. I propose a Gitbook documentation to help the community to get familiar with the polkadot universe, especially by helping them to stake. 
I also set up a monthly drop of NFT's to reward my active nominators (see my social networks).

### Description en Français :
Passionné par l'univers de Ternoa depuis le PS2 et validateur depuis les premiers blocks du Testnet, je propose à présent 3 validateurs sur le mainnet. Mes validateurs sont monitorés (24/24h, 7/7J), sécurisés pars mes soins et mis à jour régulièrement. Je propose une documentation Gitbook pour aider la communauté à se familiariser avec l'univers de polkadot notamment en les aidant à staker. 
J'ai également mis en place un drop mensuel de NFT's pour récompenser mes nominateurs actifs (voir mes réseaux sociaux).

### Useful links :
Gibook : https://crypte.gitbook.io
Twitter : https://twitter.com/_Crypte_
Matrix : @louis_crypte:matrix.org
Telegram : https://t.me/louis_crypte
Discord : Louis_Crypté#8524

**Comment by Tulleb** [comment]
### Node address:
5FXAQYymZui6nGNxCer6ShQ3jMfEdDS46aowonEYfFngJ49a

### Node Name:
Tulleb23

### Description:
Hi 👋 My name is Guillaume Bellut and I am running Tulleb23, a personal Ternoa Masternode you can join to aim for a better decentralisation.
Unlike most of the other company proposing several nodes sometimes under the same roof, I provide you a unique node which maintains the stability of Ternoa without having any dependency to other cloud computing system. Tulleb23 is a fully dedicated computer which is physically accessible at any moment; no virtual machine hosted by a third-party company around here 👍

### Descriptif:
Bonjour 👋 Je suis Guillaume Bellut, propriétaire de Tulleb23, un masternode Ternoa personnel que vous pouvez rejoindre pour aider à plus de décentralisation.
Contrairement à la plupart des autres entreprises proposant plusieurs nodes la plupart du temps sous un même toit, je ne propose qu'une unique machine afin de pousser à plus de stabilité sur la blockchain Ternoa sans avoir aucune dépendance envers d'autre entreprise de cloud computing. Tulleb23 est physiquement accessible ; pas de machine virtuelle 👍

### Useful link:
E-mail: tulleb@hotmail.com
GitHub: [Tulleb](https://github.com/Tulleb)
Stack Overflow: [Tulleb](https://stackoverflow.com/users/2854405/tulleb)
LinkedIn: [Guillaume Bellut](https://www.linkedin.com/in/tulleb/)

**Comment by ipapandinas** [comment]
### Node adress
5FWuM8Q3DRBAzu2PeqfmXqwdnegk8yiiMLXtZLB3dqJjomG8

### Node name
HERACLES

### Description in English
HERACLES is a trusted Proof-of-Stake infrastructure provider and validator to comfortably stake your CAPS and earn daily rewards. We are two french engineers: one Ternoa core team member and one experimented devops. Our Ternoa node rely on a highly secure and reliable infrastructure. The whole system is monitored 24/7 with an alerting system always up and running.
We provide some tools to facilitate the nominators' experience, such as a rewards calculator.

### Description en Français
HERACLES fournit une interface simple pour stacker confortablement ses CAPS et gagner des récompenses quotidiennes. Nous sommes une équipe de 2 ingénieurs français: un core team member de Ternoa et un ingénieur devops expérimenté. Notre nœud repose sur une infrastructure hautement sécurisée et fiable. L'ensemble du système est surveillé 24/7 avec un système d'alerte toujours opérationnel.
On propose aussi des outils pour faciliter l'expérience des nominateurs comme par exemple un calculateur de rewards.

### Useful links
Website : https://heracles.works/
Twitter : [@heracles_works](https://twitter.com/_Crypte_)
Mail: contact@heracles.works
Rewards Calculator dApp : https://calculator.heracles.works/

**Comment by cryptokyser** [comment]
### **Node adress**
5HYnwpmmvFc6Ub5SkoLKCTdgB8conmMS18JmJocBqUKG1Bf7

### **Node name**
CK-VALIDATOR

### **Description in English**
Hi there,
I am Crypto Kyser, specialist in the world of crypto-currencies and more specifically Masternodes / Mining.
This validator is completely secure, online 24 hours a day. You can delegate your tokens there serenely.
As I often say, always take a look at the project before investing (Subject, Roadmap, Discord, Tokenomics, etc.). This is essential.
Personally, I consider the Terra project to be innovative and promising.
You can watch my video (in french) which explains this project in detail at the following address: https://www.youtube.com/watch?v=QjWVkaSo654
Goodbye,
CK.

### **Description en Français**
Bonjour à tous,
Je suis Crypto Kyser, spécialiste de l'univers des crypto-monnaies et plus particulièrement du Minage / Masternodes.
Ce validateur est entièrement sécurisé, en ligne 24h/24h. Vous pouvez y déléguer vos jetons sereinement.
Comme je le dis souvent,  jetez toujours un œil sur le projet avant d'investir (Sujet, Roadmap, Discord, Tokenomics, etc.). C'est indispensable. 
Personnellement, je considère le projet Terra comme innovant et prometteur.
Vous pouvez regarder ma vidéo qui explique en détail ce projet à l'adresse suivante : https://www.youtube.com/watch?v=QjWVkaSo654
A bientôt, 
CK.

### **Useful links**
Twitter : [@cryptokyser](https://twitter.com/CryptoKyser)
Mail: [contact@cryptokyser.com](mailto:contact@cryptokyser.com)
Website  : https://www.cryptokyser.com
Ternoa video :  https://www.youtube.com/watch?v=QjWVkaSo654
